### DISCUSSION TITLE
Is there an easy way to switch over accounts or have multiple?

### DISCUSSION THREAD

**Discussion by TheMissingPort** [UltimaHoarder/UltimaScraper#910] [Q&A] [answered]
My friends have me rip stuff for them, which I don't mind but it gets a bit annoying editing the .auth.json file each time

Thank you

**Comment by nappyapp** [comment, marked as answer]
Make multiple profiles, that's why there's a .profiles directory instead of how the auth used to just be in the .settings/config.json. I would assume you just clone the "default" profile folder and rename it, and change the auth info in the new one.
